Adding Census Blocks 2020 (#4)
agrc/metadata-asset-tracking-tool: metadata/demographic/census_blocks_2020.md

# Title

<!-- The human-readable layer name that should match the pageTitle item in the data page's metadata. If it's a statewide layer, it should include "Utah", e.g. "Utah Roads" -->
Utah Census 2020 Blocks

## Brief Summary

<!-- A super-short, one-liner description of the dataset. Corresponds to the pageDescription item in the data page's metadata -->
Basic demographic data in the smallest geographic unit used by the US Census Bureau.

## Summary

<!-- A brief (<2048 characters for AGOL) explanation of the dataset to give the user a high-level overview of what the layer is when skimming lists of datasets. Will be the first description people see in Hub open data and should match the Summary section of the layer's data page  -->
Census Blocks are the smallest geographic unit used by the United States Census Bureau for tabulating population information. Due to privacy concerns, only a limited amount of data is available at the block level.

## Description

<!-- A more in-depth explanation of the dataset, where it came from and how its created, so the user can decide if its what they need. -->

### What is the dataset

> Census blocks are small geographic zones through which population data can be collected and organized. These blocks are often bounded by physical features such as roads or rivers, as well as municipal boundaries such as city limits or property lines. This dataset includes the census blocks for the State of Utah, current as of the 2020 Census and represented as polygons.

Reviewer: I can mention it in the updates, however, I believe that when the next census is released, we will be making a new layer with its own metadata for it (rather than adding the 2030 census blocks to this one). Thus, although the layer may be updated, the fact that these are the 2020 census blocks is unlikely to change, which is why I included it as part of "what is the dataset" rather than "update schedule". Does that make sense?

### What is the purpose of the dataset

Geography is a fundamental aspect of the Census, providing the framework for the once-a-decade count of population and housing. Read about some of the key geography changes in the Kem C. Gardner Policy Institute's 2020 Census Geography [Blog](https://gardner.utah.edu/blog/blog-whats-new-in-utahs-census-2020-geography/) and [fact sheet](https://d36oiwf74r1rap.cloudfront.net/wp-content/uploads/Geog-FS-Mar2021.pdf).

### What does the dataset represent

> Each Census block in this dataset is given a unique four-digit Census Block Number. For more information on how to interpret the Census Block Number, refer to this article from the [US Census Bureau](https://www.census.gov/newsroom/blogs/random-samplings/2011/07/what-are-census-blocks.html). While census blocks include information on major racial groups, housing, and population, it is important to note that census blocks are delineated by physical and municipal boundaries, not by population size. Thus, a larger block does not denote a larger population size. In fact, many census blocks contain a population number of zero (such as in the case of water-only areas).

Author: I will change that!

### How was the dataset created

> Utah's 2020 census geographic boundaries became available in February 2021 and the demographic redistricting legacy files (the 2020 Census Redistricting P.L. 94-171 Summary Files) became available on August 12, 2021. The Census Bureau will release a more user-friendly version of the 2020 Census Redistricting P.L. 94-171 Summary Files on September 30, 2021. In the meantime, UGRC, with collaboration from the Analytics Group at [WFRC](https://wfrc.org/), has worked with the August 12 release and has created ready-to-use GIS data in several formats that improve the user-friendliness of the 2020 redistricting data. Read more about these offerings in the blog post, 2020 Census Redistricting Data in Easy to Use GIS Formats

Reviewer: That paragraph is from the original data page, I kept it in because I wasn't sure if we were wanting to use it for the new metadata or not. Now that we've decided to keep the info in, I'll go ahead and change the tenses/verbage so it makes sense.

### How reliable and accurate is the dataset

> The 2020 data available on this page was downloaded from the [2020 Census Redistricting Data (P.L. 94-171)](https://www.census.gov/programs-surveys/decennial-census/about/rdo/summary-files.html#P2). All 2020 census geographies reflect boundaries as of January 1, 2020. Additional GIS files and support products are available on the [census redistricting data](https://www.census.gov/programs-surveys/decennial-census/about/rdo/summary-files.html#P2).

Reviewer 2: **suggestion**: We're on the 2020 metadata item so calling it that is a bit redundant and the on this page reference could be removed to make displaying this more generic.

> The 2020 data available on this page was downloaded

a possible replacement

> This data was downloaded

Reviewer: I will make that change!

> These datasets were downloaded from the 2020 Census Redistricting Data (P.L. 94-171) Page. All 2020 Census boundaries are current as of January 1, 2020. The Census Bureau will release the first set of corresponding demographic data in September 2021 (the 2020 Census Redistricting P.L. 94-171 Summary Files). Following that release, AGRC will append the demographic data to the existing 2020 geographies found on this page.

Reviewer 2: **suggestion**: make these singular

```suggestion
This dataset was downloaded from the 2020 Census Redistricting Data (P.L. 94-171) Page. All 2020 Census Block boundaries are current as of January 1, 2020. The Census Bureau will release the first set of corresponding demographic data in September 2021 (the 2020 Census Redistricting P.L. 94-171 Summary Files). Following that release, UGRC will append the demographic data to the existing 2020 geographies found on this page.
```

**question**: has the appending already happend? Should it be re explained?
**question**: should the current as of be displayed here again?

Reviewer: I believe the appending has already occurred. I have adjusted this paragraph accordingly, the changes will be reflected in a new pull request. Thank you for your feedback!
